src/wallet/apiHelpers.ts

```diff
@@ -12,7 +11,7 @@ class ApiError extends Error {
 
 export const getBlock = async (
   blockHash: string,
-  apiEndpoint: string = API_ENDPOINT
```

Comment:
would prefer to keep a default value in each of these helper functions as before.

Reply:
Hmm, why is that? Keeping a default feels risky—if you forget to explicitly pass the `apiEndpoint` as an input, you can end up calling an endpoint that doesn't correspond with the user's configured network. By not having a default you guarantee the correct endpoint is always used

Reply:
OK, I'm game to strike the default